uisdk/src/main/res/layout/uisdk_booking_checkout_view.xml

```diff
@@ -36,15 +36,15 @@
                 android:layout_marginStart="@dimen/kh_uisdk_spacing_grid_first_column"
                 android:layout_marginTop="@dimen/margin_16"
                 android:layout_marginEnd="@dimen/kh_uisdk_spacing_grid_first_column"
-                android:layout_marginBottom="@dimen/kh_uisdk_spacing_xsmall" />
+                android:layout_marginBottom="@dimen/margin_16" />
```

Comment:
You can use `kh_uisdk_spacing_small` 
Preferebly we better not to have a reference to the value in the name of the assets.